include/zephyr/drivers/i3c.h

```diff
@@ -464,6 +474,22 @@ struct i3c_config_controller {
 		uint32_t i2c;
 	} scl;
 
+	struct {
+		/**
+		 * Requested minimum SCL Open Drain High period in Nanoseconds
+		 *
+		 * Note:
+		 * - For the first broadcast message, spec requires tHIGH_OD >= 200 ns.
+		 * - For normal messages, tHIGH_OD must not exceed 41 ns (spec max).
+		 */
+		uint32_t high_ns;
+
+		/**
+		 * Requested minimum SCL Open-Drain LOW period in nanoseconds.
+		 */
+		uint32_t low_ns;
+	} scl_od;
```

Comment:
rename so it's more clear
```suggestion
	} scl_od_min;
```

Reply:
Updated